paybybank/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
We don't really need these files if there are no rules in them. For now we can keep them for consistency, but it looks a bit cleaner if all the comments here are removed